README.md

```diff
@@ -1,29 +1,71 @@
-# CNTK Object Recognition Video Tagging Tool
+# CNTK Object Detection Video Tagging Tool
 
-This tool provides end to end support for generating datasets for and validating Object Recognition Models with CNTK.
+This tool provides end to end support for generating datasets for and validating Object Detection Models with CNTK.
 
 It supports the following scenarios:
 
 - Computer Assisted Tagging of Objects In Video Using Custom implementation of Camshift Algorithm
```

Comment:
For Camshift, call it "of the Camshift tracking algorithm"
Also, add a link to the original paper:http://opencv.jp/opencv-1.0.0_org/docs/papers/camshift.pdf
E.g, make "Camshift tracking algorithm" a hyper link.